### PR TITLE
Parse product ID

### PR DESCRIPTION
**Pull request by twzkraus**
- Removes example data 
- Changes anchor tag for database seeding script
- adds CORS to server
- uses item.id for keys in React components

### CODE REVIEW

**Comment by twzkraus** [comment]
Note: CircleCI test failure related to recent changes to the DOM. I may go back and modify the tests later on.

**Comment by twzkraus** [comment]
@michaelrtobias Thanks!

File: src/RecommendedProducts.jsx [hunk 1]
```diff
@@ -29,8 +31,8 @@ const RecommendedProducts = ({totalItems, itemsShown, productId, heading}) => {
   }
 
   useEffect(() => {
-    if (productId) {
-      axios.get(`/products/id/${productId}`)
+    if (Number.isInteger(parseInt(productId))) {
```
Comment (michaelrtobias) [inline src/RecommendedProducts.jsx:34]: Smart making sure it's a number. I did the same.

**Review by michaelrtobias — APPROVED** [review]
Seems fine to me!

**Pull request merged**